articles/confidential-computing/faq.md

```diff
@@ -27,32 +27,35 @@ If your Azure issue is not addressed in this article, visit the Azure forums on
+   No. The virtual machines can only be deployed on Generation 2 virtual machines with Ubuntu Server 18.04, Ubuntu Server 16.04, and Windows Server 2016 Datacenter. Read more about Gen 2 VMs on [Linux](../virtual-machines/linux/generation-2.md) and [Windows](../virtual-machines/windows/generation-2.md)
 
 1. **DCsv2 virtual machines are grayed out in the portal and I can't select one**
 
-   Based on the information bubble next to the VM, there are different actions to take:
+   Make sure to toggle the filters as listed below and see if they become available:
     -	**UnsupportedGeneration**: Change the generation of the virtual machine image to “Gen2”.
     -	**NotAvailableForSubscription** : The region isn't yet available for your subscription. Select an available region.
-    -	**InsufficientQuota**: [Create a support request to increase your quota](../azure-portal/supportability/per-vm-quota-requests.md). Free trial subscriptions don't have quota for confidential computing VMs. 
+    
+1. **I get an ARM deployment failure error "Operation could not be completed as it results in exceeding approved standard DCSV2 Family Cores Quota**
+
+-	**InsufficientQuota**: [Create a support request to increase your quota](../azure-portal/supportability/per-vm-quota-requests.md). Free trial subscriptions don't have quota for confidential computing VMs. 
 
 1. **DCsv2 virtual machines don't show up when I try to search for them in the portal size selector**
 
    Make sure you've selected an available region. Also make sure you select “clear all filters” in the size selector. 
 
 1. **What’s the difference between DCsv2-Series and DC-Series VMs?**
 
-   DC-Series VMs run on older 6-core Intel Processors with SGX. These have less total memory, less EPC (Enclave Page Cage) memory, and are available in less regions. These VMs are only available in US East and Europe West are available in two sizes: Standard_DC2s and Standard_DC4s. They will not go GA and can only be deployed in the [Confidential Compute DC-Series VM [Preview]](https://azuremarketplace.microsoft.com/marketplace/apps/microsoft-azure-compute.confidentialcompute?tab=Overview) Marketplace instance
+   DC-Series VMs run on older 6-core Intel Processors with SGX and have less memory, smaller EPC (Enclave Page Cage) memory, and available in less number of regions(US East and Europe West in Standard_DC2s and Standard_DC4s sizes). These VM's are not planned to be made Generally Available and not recommended. If you still like to deploy these VM's please use this Azure Market place template - [Confidential Compute DC-Series VM [Preview]](https://azuremarketplace.microsoft.com/marketplace/apps/microsoft-azure-compute.confidentialcompute?tab=Overview) Marketplace instance
```

Comment:
Should write as "Intel SGX". 
"These VM's are not planned to be made Generally Available and not recommended". Grammar fix to make VM's plural (VMs) not possessive (VM's). 
Try to avoid words like "please" to ensure the directions are succinct and direct. Also make sure Marketplace is one word.

Reply:
grammer, all agreed